content/yaml-publishing/steam.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Steam
+description: How to deploy an app to Steam using codemagic.yaml
+weight: 5
+---
+
+Codemagic allows you to deploy your unity application to Steam.
+
+{{<notebox>}}
+This guide only applies to workflows configured with the **codemagic.yaml**.
+{{</notebox>}}
+
+## Prerequisites
+
+- A Unity **Plus** or **Pro** license. Your license is used to activate Unity on the Codemagic build server so the project can be built. The license is returned automatically once the workflow completes.
+- A Steam Partner account is required to publish to Steam.
+
+## Getting Started
+
+SteamCMD is the tool used to upload builds to Steam. SteamCMD requires logging into Steam and will normally require a Steam Guard code be entered.
+To solve this problem, there are two options.
+
+1. Disable Steam Guard for the account doing the Steam upload.  This is not recommended, as it makes the Steam account less secure.
+
+2. Use sentry files that are generated after logging in successfully to Steam on the build machine. When these sentry files are present on the build machine, a Steam Guard code is not required.
+So we are going to save the sentry files as secure environment variables and then place them at the correct path when the build starts.
```

Comment:
Thus, we will save the sentry files as secure environment variables and place them at the correct path when the build starts.